.github/workflows/release.yml

```diff
@@ -43,21 +43,20 @@ jobs:
           # Ensure that there is no existing kapp installed
           rm -f /tmp/bin/kapp
 
-      - name: Create kapp binary
+      - name: Run Test cases
```

Comment:
Should we also move this step to the end? The tests can fail due to various reasons and sometimes we might wanna go ahead with the release even if the tests have failed, so if we move this step to the end then we will know if the checksums are matching. But even if the tests fail and the remaining steps are still executed, then I guess it's fine to have it here itself.